src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import lotto.enums.LottoRank;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+    Map<LottoRank, Integer> winningCount = new HashMap<>();
+    double profitRate;
+
+    public LottoResult(Lottos lottos, Lotto winningLotto) {
+        updateWinningCount(lottos, winningLotto);
+        updateProfitRate(lottos.size());
+    }
+
+    private void updateWinningCount(Lottos lottos, Lotto winningLotto) {
+        for (Lotto lotto : lottos) {
+            LottoRank rank = LottoRank.matchRank(lotto.countMatchedNumbers(winningLotto));
+            winningCount.put(rank, winningCount.getOrDefault(rank, 0) + 1);
+        }
+    }
+
+    private void updateProfitRate(int lottoCount) {
+        if(lottoCount == 0){
```

Comment:
`if`에 대한 컨벤션이 들쑥날쑥 한데요 😅 
이 부분 한변 신경써주시면 감사하겠습니다 🙇 🙏 

```suggestion
        if (lottoCount == 0) {
```
그리고 `0`도 매직넘버이니 상수로 추출 되면 좋을 것 같습니다 🙏